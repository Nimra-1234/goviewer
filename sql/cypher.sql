CREATE EXTENSION agensgraph;

SET search_path TO ag_catalog;

SELECT * FROM cypher(NULL) AS r(c text);
SELECT * FROM cypher($$RETURN 0$$) AS r(c text);
