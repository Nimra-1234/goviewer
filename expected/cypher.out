CREATE EXTENSION agensgraph;
SET search_path TO ag_catalog;
SELECT * FROM cypher(NULL) AS r(c text);
ERROR:  unhandled cypher(text) function call
DETAIL:  NULL
SELECT * FROM cypher($$RETURN 0$$) AS r(c text);
ERROR:  unhandled cypher(text) function call
DETAIL:  RETURN 0
